### PR TITLE
FB MAJ guide cPanel on VPS

### PR DESCRIPTION
**Pull request by fabienOVH**
Update of this guide to provide more information to the user

### CODE REVIEW

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
 title: 'Déployer cPanel sur un VPS'
 excerpt: 'Découvrez comment instancier un VPS avec l’application cPanel pré-installée.'
-updated: 2021-10-14
+updated: 2024-01-08
 ---
 
 ## Objectif
 
-cPanel est un panneau de configuration conçu pour les hébergeurs web. Constitué d'une interface graphique permettant l'automatisation des paramètres, l'hébergement de site web est ainsi simplifié.
+cPanel est un panneau de configuration qui simplifie la gestion d'hébergement web, rendant les tâches complexes accessibles même aux utilisateurs ayant peu de connaissances techniques. Il offre une vaste gamme de fonctionnalités, y compris la gestion des emails, des domaines, des bases de données et de la sécurité. Constitué d'une interface graphique permettant l'automatisation des paramètres, l'hébergement de sites web est ainsi simplifié.
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
cPanel est un panneau de configuration qui facilite la gestion d'hébergements web. Il rend les tâches complexes plus accessibles y compris pour les nouveaux utilisateurs. Il offre une vaste gamme de fonctionnalités comme, par exemple, pour la gestion : 

- des e-mails, 
- des domaines,
- des bases de données,
- de la sécurité,
- etc.

Constitué d'une interface graphique permettant l'automatisation des paramètres, l'hébergement de sites web est ainsi simplifié.
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
 
 **Découvrez comment déployer cPanel avec les applications pré-installées sur un VPS.**
 
 ## Prérequis
 
-- Un [VPS de la gamme actuelle](https://www.ovhcloud.com/fr/vps/){.external} (offres Value, Essential, Comfort ou Elite).
+- Posséder un [VPS de la gamme actuelle](https://www.ovhcloud.com/fr/vps/){.external} (offres Value, Essential, Comfort ou Elite).
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
- Disposer d'une offre [VPS récente](https://www.ovhcloud.com/fr/vps/){.external} (offres Value, **Essential**, **Comfort** ou **Elite**).
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
 
 ```
  |    Vos application(s):
- |    Vous pouvez vous connecter à cpanel depuis https://<ip>:2087/<session_parameters>
+ |    Application: cpanel
+ |    Vous pouvez vous connecter à cpanel depuis https://<nomduserveur>:2087/<session_parameters>
 ```
 
-Si vous disposez déjà d'un VPS et que vous souhaitez y installer cPanel, vous pouvez réinstaller le VPS à partir de votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) à l'aide du modèle « CentOS 7 - cPanel » (disponible uniquement avec une offre VPS compatible).
+Si vous disposez déjà d'un VPS et que vous souhaitez y installer cPanel, vous pouvez réinstaller le VPS à partir de votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) via un [OS compatible avec cPanel](https://www.ovhcloud.com/en/vps/os/).
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
Si vous disposez déjà d'un VPS et que vous souhaitez y installer cPanel, vous pouvez réinstaller le VPS à partir de votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) via un [OS compatible avec cPanel](https://www.ovhcloud.com/fr/vps/os/).
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 4]
```diff
@@ -37,18 +38,13 @@ Si vous disposez déjà d'un VPS et que vous souhaitez y installer cPanel, vous
 
 ### Première connexion
 
-Une fois que vous avez reçu l'e-mail avec le lien unique, cliquez sur ce lien pour effectuer la configuration initiale.
+Une fois que vous avez reçu l'e-mail avec le lien unique, cliquez sur ce lien pour effectuer la configuration initiale. Si le lien a déjà expiré, connectez-vous en SSH au serveur et exécutez la commande « sudo whmlogin » pour générer un nouveau lien.
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
Une fois que vous avez reçu l'e-mail contenant le lien unique, cliquez sur ce lien pour effectuer la configuration initiale. Si le lien a déjà expiré, connectez-vous en SSH au serveur et exécutez la commande « sudo whmlogin » pour générer un nouveau lien.
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 5]
```diff
@@ -37,18 +38,13 @@ Si vous disposez déjà d'un VPS et que vous souhaitez y installer cPanel, vous
...
 
-> [!primary]
->
-> Si le lien a déjà expiré, connectez-vous en SSH au serveur à l’aide de l’utilisateur CentOS et exécutez la commande « sudo whmlogin » pour générer un nouveau lien.
->
+L'URL générée par la commande « sudo whmlogin » vous permet de vous connecter sans informations d'identification (utilisateur et mot de passe) à votre interface WHM. WHM est une surcouche de cPanel. Vous pourrez accéder à cPanel après avoir effectué les étapes suivantes.
 
-L'URL citée plus haut vous permet de vous connecter sans informations d'identification (utilisateur et mot de passe) à votre interface WHM.
+#### Étape 1 : accéptez les conditions d'utilisation de cPanel
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
#### Étape 1 : accepter les conditions d'utilisation de cPanel
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 6]
```diff
@@ -62,7 +58,37 @@ Indiquez les serveurs de messagerie et de noms (nameservers) que vous souhaitez
 
 ![cPanel](images/change_root.png){.thumbnail}
 
-Vous devriez à présent pouvoir vous connecter à WHM et en SSH en utilisant l'utilisateur root avec le mot de passe qui vient d'être défini.
+Vous pouvez maintenant vous connecter à votre serveur en SSH en utilisant l'utilisateur root avec le mot de passe que vous venez de définir.
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
Désormais, vous pouvez vous connecter à votre serveur en SSH en utilisant l'utilisateur root avec le mot de passe que vous venez de définir.
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 7]
```diff
@@ -62,7 +58,37 @@ Indiquez les serveurs de messagerie et de noms (nameservers) que vous souhaitez
...
+
+Vous êtes redirigé vers votre interface cPanel.
+
+![cPanel](images/manager_cpanel.png){.thumbnail}
+
+Vous pouvez désormais utiliser cPanel. Pour toute information complémentaire concernant cPanel, consultez la [documentation officielle](https://docs.cpanel.net/).
+
+> [!primary]
+>
+> Dans la barre de navigation de votre navigateur, entrez les URL suivantes pour vous connecter à :
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
> Dans la barre de navigation de votre navigateur, entrez les URLs suivantes pour vous connecter à :
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 8]
```diff
@@ -62,7 +58,37 @@ Indiquez les serveurs de messagerie et de noms (nameservers) que vous souhaitez
...
+> - cPanel : https://<IP_V4>:2083/ (utilisez les identifiants qui viennent d'être créés dans l'interface WHM)
+> - WHM : https://<IP_V4>:2087/ (utilisez le nom d'utilisateur « root » ainsi que le mot de passe reçu dans l’email d’achat du service ou le mot de passe SSH qui a été modifié dans l’interface WHM)
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
> - WHM : https://<IP_V4>:2087/ (utilisez le nom d'utilisateur « root » ainsi que le mot de passe reçu dans l’e-mail d’achat du service ou le mot de passe SSH qui a été modifié dans l’interface WHM)
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 9]
```diff
@@ -62,7 +58,37 @@ Indiquez les serveurs de messagerie et de noms (nameservers) que vous souhaitez
...
+>
+> Retrouvez votre adresse IPv4 dans le mail que vous avez reçu suite à la commande de votre VPS avec la > distribution cPanel.
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md]:
```suggestion
> Retrouvez votre adresse IPv4 dans le mail que vous avez reçu suite à la commande de votre VPS avec la distribution cPanel.
```

File: pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md [hunk 10]
```diff
@@ -62,7 +58,37 @@ Indiquez les serveurs de messagerie et de noms (nameservers) que vous souhaitez
...
+>
+
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/cpanel/guide.fr-fr.md:100]:
```suggestion
```

**Review by Kazarus4 — COMMENTED** [review]
Some screens are still in French, you must redo in English.
don't forget to hide some infos (ips, hostname, etc too)

**Comment by OVH-Team-Guides** [comment]
Thank you for applying suggestions @fabienOVH 
Please update manager_cpanel.png so that it's in English and sensitive information (IPs etc) are hidden.

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @fabienOVH

**Pull request merged**